repo/packages/J/jenkins/17/resource.json

```diff
@@ -7,7 +7,7 @@
   "assets": {
     "container": {
       "docker": {
-        "jenkins-302-2322": "mesosphere/jenkins:3.0.2-2.32.2"
+        "jenkins-302-2322": "mesosphere/jenkins:3.0.3-2.32.3"
```

Comment:
This should be `"jenkins-303-2323"` to match the `image` resource in the Marathon template.